components/Insurance/Pools/index.tsx

```diff
@@ -42,7 +42,7 @@ type CProps = {
 const HealthCell: React.FC<CProps> = styled(({ pool, className }: CProps) => {
     return (
         <div className={className}>
-            <SProgressBar percent={pool?.health.toNumber()} />
+            <SProgressBar percent={parseFloat(pool?.health.toFixed(2))} />
```

Comment:
this could have just been done once in progressBar

Reply:
It makes sense not to have anything functionality wise in ProgressBar so I'm keeping this